Examples/ObjectiveC/RuntimeAddLineExample.m

```diff
@@ -52,10 +52,12 @@ - (void)drawPolyline:(NSData *)geoJson {
     layer.lineCap = [MGLStyleValue valueWithRawValue:[NSValue valueWithMGLLineCap:MGLLineCapRound]];
     layer.lineColor = [MGLStyleValue valueWithRawValue:[UIColor colorWithRed:59/255.0 green:178/255.0 blue:208/255.0 alpha:1]];
     // Use a style function to smoothly adjust the line width from 2pt to 20pt between zoom levels 14 and 18. The `interpolationBase` parameter allows the values to interpolate along an exponential curve.
-    layer.lineWidth = [MGLStyleValue valueWithInterpolationBase:1.5 stops:@{
-        @14: [MGLStyleValue valueWithRawValue: @2],
-        @18: [MGLStyleValue valueWithRawValue: @20]
-    }];
+    layer.lineWidth = [MGLStyleValue valueWithInterpolationMode:MGLInterpolationModeExponential
+        cameraStops:@{
+                      @14: [MGLStyleValue valueWithRawValue: @2],
```

Comment:
Let’s keep the indentation the same as it was (i.e., more to the same) — may prevent line-wrapping issues.